docs/LogGuide.md

```diff
@@ -2821,3 +2853,399 @@ Parsed Log Line Examples:
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=CountdownCancel&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+269|[timestamp]|[id]|[worldId]|[name]
+
+Parsed Log Line Structure:
+[timestamp] 269 10D:[id]:[worldId]:[name]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>269)\|(?<timestamp>[^|]*)\|(?<id>[^|]*)\|(?<worldId>[^|]*)\|(?<name>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 269 (?<type>10D):(?<id>[^:]*):(?<worldId>[^:]*):(?<name>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+269|2024-02-29T15:19:55.3490000-08:00|10FF0001|0036|Tini Poutini|e17efb9d120adea0
+269|2024-02-29T15:34:22.8940000-08:00|10FF0002|0036|Potato Chippy|e17efb9d120adea0
+
+Parsed Log Line Examples:
+[15:19:55.349] 269 10D:10FF0001:0036:Tini Poutini
+[15:34:22.894] 269 10D:10FF0002:0036:Potato Chippy
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=CountdownCancel&lang=en-US) -->
+
+<a name="line270"></a>
+
+### Line 270 (0x10E): ActorMove
+
+An `ActorMove` packet is sent to instruct the game client to move an actor to a new position
+(for example, in UWU, when Titan turns prior to jumping to indicate the direction of his jump).
+The FFXIV client interpolates the actor's movement between the current position and the new position.
+Currently, these log lines are emitted only for non-player actors (id >= 0x40000000).
+
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=ActorMove&lang=en-US) -->
+
+#### Structure
+
+```log
+Network Log Line Structure:
+270|[timestamp]|[id]|[heading]|[?]|[?]|[x]|[y]|[z]
+
+Parsed Log Line Structure:
+[timestamp] 270 10E:[id]:[heading]:[?]:[?]:[x]:[y]:[z]
+```
+
+#### Regexes
+
+```log
+Network Log Line Regex:
+^(?<type>270)\|(?<timestamp>[^|]*)\|(?<id>[^|]*)\|(?<heading>[^|]*)\|(?:[^|]*\|){2}(?<x>[^|]*)\|(?<y>[^|]*)\|(?<z>[^|]*)\|
+
+Parsed Log Line Regex:
+(?<timestamp>^.{14}) 270 (?<type>10E):(?<id>[^:]*):(?<heading>[^:]*)(?::[^:]*){2}:(?<x>[^:]*):(?<y>[^:]*):(?<z>[^:]*)(?:$|:)
+```
+
+#### Examples
+
+```log
+Network Log Line Examples:
+270|2024-03-02T13:14:37.0430000-08:00|4000F1D3|-2.2034|0002|0014|102.0539|118.1982|0.2136|4601ae28c0b481d8
+270|2024-03-02T13:18:30.2960000-08:00|4000F44E|2.8366|0002|0014|98.2391|101.9623|0.2136|2eed500a1505cb03
+270|2024-03-02T13:18:30.6070000-08:00|4000F44E|-2.5710|0002|0014|98.2391|101.9318|0.2136|51bc63077eb489f3
+
+Parsed Log Line Examples:
+[13:14:37.043] 270 10E:4000F1D3:-2.2034:0002:0014:102.0539:118.1982:0.2136
+[13:18:30.296] 270 10E:4000F44E:2.8366:0002:0014:98.2391:101.9623:0.2136
+[13:18:30.607] 270 10E:4000F44E:-2.5710:0002:0014:98.2391:101.9318:0.2136
+```
+
+<!-- AUTO-GENERATED-CONTENT:END (logLines:type=ActorMove&lang=en-US) -->
+
+<a name="line271"></a>
+
+### Line 271 (0x10F): ActorSetPos
+
+An `ActorSetPos` packet is sent to instruct the game client to set the position of an actor
+with no interpolated movement (for example, in UWU, to set the location of the Ifrit clones).
```

Comment:
It may be worth noting that sometimes these `ActorSetPos` lines are accompanied by other data (in other log lines or network packets) indicating that an animation should be played if the actor is already visible.

This could for example be something like:
1. Ability use with animation tied to it, for example to make the actor appear to be jumping
2. A few seconds later, `ActorSetPos` is sent to move actor
3. Ability use or other network packet to indicate an animation should be played, for example to make the actor appear to be landing